front/pages/w/[wId]/ds/[name]/settings.tsx

```diff
@@ -172,9 +209,18 @@ export default function DataSourceSettings({
 function StandardDataSourceSettings({
   owner,
   dataSource,
+  handleUpdate,
+  isUpdating,
 }: {
   owner: WorkspaceType;
   dataSource: DataSourceType;
+  handleUpdate: (settings: {
```

Comment:
This type is not coherent with the type of handle update? that's a bit surprising?

Reply:
To address this I adapted the callee signature and explicited the types in the component calls
Note that it is actually not necessary to explicit the types, I guess because TS sees they are compatible? 
Since the types are already very clear both options (expliciting or not expliciting) are okay to me so if you have a preference tell me otherwise we can keep it as is